src/atoms/typography/heading.test.tsx

```diff
@@ -1,5 +1,32 @@
 import React from "react";
+import faker from "faker";
+import { Heading } from "./heading";
+import { render } from "@testing-library/react";
 
 describe("Heading", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/14", () => {});
+    test("when default props, renders heading", () => {
+        // Assert
+        const expected = faker.random.words();
+
+        // Act
+        const { getByText } = render(<Heading>{expected}</Heading>);
+
+        // Assert
+        expect(getByText(expected)).not.toBeNil();
+    });
+
+    test("when given cssClassName prop, renders with given class name", () => {
```

Comment:
Recommend naming this `when cssClassName prop has value, renders with class name`

Reply:
I updated the tests if you want to take a second look. 👍